java/c/src/test/java/org/apache/arrow/c/RoundtripTest.java

```diff
@@ -770,6 +771,17 @@ public void testStructVector() {
     }
   }
 
+  @Test
+  public void testRunEndEncodedVector() {
```

Comment:
shall we also add a test for an empty REE Vector?

Reply:
sure, see testEmptyRunEndEncodedVector